exchanges/binance/binance_websocket.go

```diff
@@ -662,13 +683,18 @@ func (b *Binance) processJob(p currency.Pair) error {
 	if err != nil {
 		errClean := b.Websocket.Orderbook.FlushOrderbook(p, asset.Spot)
 		if err != nil {
```

Comment:
Do you mean to chack` errClean` here? if there is an err, this will already be true and will log with a potential nil

Reply:
Yeah, nice pickup. Pushed an update.